NetKet/Machine/abstract_layer.hpp

```diff
@@ -114,7 +102,7 @@ class AbstractLayer {
   */
   virtual void UpdateLookup(const VectorType &input,
```

Comment:
Maybe we can just call these UpdateOutput, just to remove any reference to Lookup tables ?

Reply:
sure.